llvm/test/Feature/Repo/repo_hash_structType_name.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: rm -f %t.db
+; RUN: env REPOFILE=%t.db opt -S -mtriple x86_64-pc-linux-gnu-repo %s | FileCheck %s
+
+target triple = "x86_64-pc-linux-gnu-elf"
+
+%"class.clang::SourceLocation" = type { i32 }
+
+%"class.clang::OMPClause" = type { %"class.clang::SourceLocation", %"class.clang::SourceLocation", i32 }
+%"class.clang::OMPReadClause" = type { %"class.clang::OMPClause" }
+%"class.clang::OMPWriteClause" = type { %"class.clang::OMPClause" }
+
+define %"class.clang::OMPClause"* @_ZN5clang13TreeTransformIN12_GLOBAL__N_120TemplateInstantiatorEE22TransformOMPReadClauseEPNS_13OMPReadClauseE(%"class.clang::OMPReadClause"* readnone) align 2 {
```

Comment:
Can we simplify some of these names? It might make the test easier to understand.

Reply:
Yes, I have simplified the test. Thanks.